mobify/bellows
# Destroy

Adding a destroy function similar to other plugins, and updating tests.

Status: **Ready to merge**

Reviewers: @tedtate 

## Changes
- Added `destroy` function
- Updated tests to use `destroy`

## How to test-drive this PR
- checkout this branch
- `grunt test`
- `grunt serve` and make sure stuff isn't broken
